ui/app/AppLayouts/Profile/views/SettingsContentBase.qml

```diff
@@ -120,6 +120,12 @@ Item {
                         easing.type: Easing.InOutQuad
                     }
                 }
+
+                Rectangle {
+                    color: "red"
```

Comment:
What's the purpose of this, and shouldn't it be using the "danger" color instead?

Reply:
Oops, this debug code got here by mistake. Removed. Thanks!